apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -164,7 +163,7 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
     this._terminalProvider.verboseEnabled = this.isDebug;
 
     try {
-      this._normalizeCwd();
+      await this._normalizeCwdAsync();
```

Comment:
```suggestion
      this._normalizeCwd();
```
No longer needs to be async